File: Node/Matrizloca.js
const matrixLoca = () => {
  let input = '';
  let size = '';
  let matrix = [];

  process.stdin.setEncoding('utf8');

  process.stdin.on('data', chunk => {
    input += chunk;
  });

  process.stdin.on('end', () => {
    input = input.trim().split('\n');

    size = input.shift().split(' ');

    const x = size[0];
    const y = size[1];

    for (let i = 0; i < x; i++) {
      matrix[i] = input[i].split(' ').map(value => Number.parseInt(value));
    }

    for (let i = 0; i < x; i++) {
      for (let j = 0; j < y; j++) {
        if (i % 2 == 0) matrix[i][j] += 1;
        if (j % 2 == 0) matrix[i][j] += 2;
        if (i % 2 != 0 && j % 2 != 0) matrix[i][j] -= 3;
      }
    }

    matrix.forEach(row => {
      console.log(row.join(' '));
    });
  });
}

matrixLoca();

File: Python/MatrizLoca.py
import sys

def matrizLoca():
    input_lines = sys.stdin.read().split('\n')

    size = input_lines[0].split()
    x = int(size[0])
    y = int(size[1])

    matrix = []

    for i in range(x):
        row = input_lines[i+1].split()
        matrix.append(list(map(int, row)))

    for i in range(x):
        for j in range(y):
            if i % 2 == 0:
                matrix[i][j] += 1
            if j % 2 == 0:
                matrix[i][j] += 2
            if i % 2 != 0 and j % 2 != 0:
                matrix[i][j] -= 3

    for row in matrix:
        print(' '.join(map(str, row)))

matrizLoca()